src/style.css

```diff
@@ -0,0 +1,87 @@
+body {
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  margin-top: 10px;
+  margin-bottom: 20px;
```

Comment:
apply `margin-bottom` to `.form` instead of `.input`

You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children